Esri/arcgis-maps-sdk-swift-samples
# [New] Create mobile geodatabase

## Description

This PR implements `Create mobile geodatabase` in `Edit and Manage Data` category.
URL to README: [README](https://github.com/Esri/arcgis-maps-sdk-swift-samples/tree/Caleb/New-CreateMobileGeodatabase/Shared/Samples/Create%20mobile%20geodatabase)

## Linked Issue(s)

- `swift/issues/4607`

## How To Test

- Tap on the map to add feature points.
- Tap "View Table" to see the features in the feature table.
- Tap the share button to export the ".geodatabase" file, e.g. tap "Save to Flies"-> "Save".

## Screenshots

![create-mobile-geodatabase-1](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/34120440/e01bb57a-b414-4fb1-8b7f-bc0b10e1bd84) ![create-mobile-geodatabase-2](https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/34120440/199e25db-a409-4989-9139-5b5a047bce94)
